pkg/reconciler/openshift/rbac/rbac.go

```diff
@@ -274,3 +315,41 @@ func createSA(ctx context.Context, saInterface v1.ServiceAccountInterface, ns st
 
 	return sa, nil
 }
+
+func createTrustedCABundleConfigMap(ctx context.Context, cfgInterface v1.ConfigMapInterface, name, ns string) error {
+	c := &corev1.ConfigMap{
```

Comment:
Why are we not reading the configmap yaml?

Reply:
Because that is for one time creation in `openshift-pipelines` namespace in `PreReconcile` phase while this is for all the other namespaces.